api/envoy/config/filter/http/aws_lambda/v2alpha/aws_lambda.proto

```diff
@@ -42,6 +42,17 @@ message Config {
 
   // Determines the way to invoke the Lambda function.
   InvocationMode invocation_mode = 3 [(validate.rules).enum = {defined_only: true}];
+
+  // Indicates that before signing headers, the host header will be swapped with
+  // this value. If not set or empty, the original host header value
+  // will be used and no rewrite will happen.
+  //
+  // Note: this rewrite affects both signing and host header forwarding. However, this
+  // option shouldn't be used with
+  // :ref:`HCM host rewrite <envoy_api_field_route.RouteAction.host_rewrite>` given that the
+  // value set here would be used for signing whereas the value set in the HCM would be used
+  // for host header forwarding which is not the desired outcome.
+  string host_rewrite = 4;
```

Comment:
I wasn't sure if I had to change this version or just v3. if not needed I can just revert this change

Reply:
we can ignore this proto and not make any changes here